package.json

```diff
@@ -16,6 +16,11 @@
   "author": "EasyMeet Team",
   "license": "MIT",
   "dependencies": {
+    "@devexpress/dx-react-scheduler-material-ui": "^2.7.4",
```

Comment:
This should be removed. Try doing a git pull, then resubmitting the pull request.